src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -34,27 +34,28 @@ const Feature: React.FC<FeatureProps> = props => {
   const video2 = injectedData.video_2
 
   const showRails =
-    featuredRails?.collections_rail?.items?.length ||
-    featuredRails?.auctions_rail?.items?.length ||
-    featuredRails?.fairs_rail?.items?.length
+    !!featuredRails?.collections_rail?.items?.length ||
```

Comment:
These (and similar changes) are to enable the edge case of us not having content in the JSON file for one or more rails.

There's also a slight performance win in that we're only fetching for the rails data at all if we have data for it (via an `@include(if:...` directive.